src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,9 +1,11 @@
+
 package core.mate.academy.model;
 
 /**
  * Add some custom fields that could be only in Truck
  * Do not remove no-field constructor
  */
+
 public class Truck extends Machine {
```

Comment:
The class `Truck` should contain custom fields that represent the state of a truck. For example, you might have fields such as `loadCapacity`, `engineType`, etc. Make sure to add these fields with appropriate access modifiers and methods (getters/setters).